ios/FluentUI/Vnext/Divider/Divider.swift

```diff
@@ -47,8 +50,10 @@ public struct FluentDivider: View {
 
         return Rectangle()
             .fill(Color(state.color ?? tokens.color))
-            .frame(maxWidth: isHorizontal ? .infinity : thickness,
-                   maxHeight: isHorizontal ? thickness : .infinity)
+            .frame(minWidth: state.thickness,
+                   maxWidth: isHorizontal ? .infinity : state.thickness,
+                   minHeight: state.thickness,
+                   maxHeight: isHorizontal ? state.thickness : .infinity)
```

Comment:
This feels overwrought compared to our previous approach:

```
.frame(width: isHorizontal ? nil : state.thickness,
       height: isHorizontal ? state.thickness : nil)
```

Reply:
It is necessary that the minWidth and minHeight are specified with the default thickness.
The maxHeight is how we ensure the SwiftUI View will always confirm to its containing UIView.
It's okay to have both max and min set to the same value.

Reply:
Both seem to work.